apps/paymaster-proxy/src/middlewares/getRateLimiter.ts

```diff
@@ -0,0 +1,16 @@
+import { rateLimit } from 'express-rate-limit'
+import type { Redis } from 'ioredis'
+import { RedisStore } from 'rate-limit-redis'
+
+export const getRateLimiter = (redisClient: Redis) =>
+  rateLimit({
+    windowMs: 5 * 60 * 1000, // 5 minutes
+    limit: 100,
+    standardHeaders: 'draft-7',
+    legacyHeaders: false,
+    store: new RedisStore({
+      // see https://github.com/express-rate-limit/rate-limit-redis
+      // @ts-expect-error - Known issue: the `call` function is not present in @types/ioredis
```

Comment:
i know this is from the issue theres probably ways around this in the future, i think the args basically get lumped into this  for ioredis `[command, key, value]` where value can be anything